include/xrpl/protocol/UintTypes.h

```diff
@@ -61,8 +61,7 @@ using NodeID = base_uint<160, detail::NodeIDTag>;
 /** MPTID is a 192-bit value representing MPT Issuance ID,
  * which is a concatenation of a 32-bit sequence (big endian)
  * and a 160-bit account */
-// TODO - edhennis - Add a tag
-using MPTID = uint192;
+using MPTID = base_uint<192>;
```

Comment:
`uint192` is defined as `base_uint<192>`, so this change does nothing. What I was referring to in the comment to "Add a tag" is more like:
```
/** A 160-bit unsigned that uniquely identifies an account. */
using AccountID = base_uint<160, detail::AccountIDTag>;
```
That would make `MPTID` something distinct from a `uint192` to prevent accidentally mixing types, though explicit and implicit conversions are available when needed.

Reply:
Adding a tag is going to be somewhat involved since it'll require defining new `ST` class, something like `STMptID`, plus whatever other required code to support this new type.

Reply:
as @gregtatcam said; also it is not strictly needed for SAV. I would rather see it in a separate PR.

Reply:
> Adding a tag is going to be somewhat involved since it'll require defining new `ST` class, something like `STMptID`, plus whatever other required code to support this new type.

That's why it was in there as a TODO. 😄

Reply:
I guess I shouldn't have removed that TODO :cry: